minidns-core/src/main/java/org/minidns/record/SVCB.java

```diff
@@ -0,0 +1,102 @@
+package org.minidns.record;
+
+import org.minidns.dnsname.DnsName;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * SVCB Record Type (Service binding)
+ *
+ * https://tools.ietf.org/html/draft-ietf-dnsop-svcb-httpssvc-01
+ */
+class SVCB extends RRWithTarget {
+
+    /**
+     * The priority indicates the SvcRecordType.
+     * https://tools.ietf.org/html/draft-ietf-dnsop-svcb-httpssvc-01#section-2.4
+     */
+    public final int priority;
+
+    /**
+     * SvcFieldValue
+     * A set of key=value pairs.
+     * https://tools.ietf.org/html/draft-ietf-dnsop-svcb-httpssvc-01#section-2.1
+     */
+    public final Map<String, String> values;
+
+    // The first group is the key. They key can only be a-z, 0-9 or "-"
+    // The second group is the value. It can be a lot of things (see https://tools.ietf.org/html/draft-ietf-dnsop-svcb-httpssvc-01#section-2.1.1)
+    //    except for DQUOTE (hence it can be excluded from the regex-group)
```

Comment:
nit: superfluous whitespace at the front

Reply:
It's meant as an indent for the previous line (which would be too long as a single line)